### PR TITLE
feat: add social buttons on confirmations modals

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-registry#1492

- add a [ShareSection](https://github.com/regen-network/regen-web/pull/1772/files#diff-ef1ac4dc6afba71474bd66daf2fd7be2fd51c8d5e770241858d01d62f003c492) component
- use it on buy and retire confirmation modals

image

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood`, `hambach` and optionally `master` (see below)

#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

1. https://deploy-preview-1772--regen-registry.netlify.app/storefront
2. test buy actions (from storefront or project card) and retire action (from portfolio)
3. check social buttons at the bottom of the confirmation modal

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by flagrede** [comment]
@erikalogie You can test the new share buttons [here](https://deploy-preview-1772--regen-registry.netlify.app/storefront) on the following actions:
- buy actions (from storefront or project card)
- retire action (from portfolio)

Note that currently, all links target the home page because the OG preview is only working for the home for now. If we want to change some of these links even if the OG preview does not match we can do it.

File: web-registry/src/features/marketplace/BuySellOrderFlow/BuySellOrderFlow.constants.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const BUY_FLOW_TWITTER_TEXT =
```
Comment (wgwz) [inline web-registry/src/features/marketplace/BuySellOrderFlow/BuySellOrderFlow.constants.ts:1]: could we make this a regular string and have it wrapped in a function url encodes it? that would make the text a little easier to change

**Review by wgwz — APPROVED** [review]
Approving; just added a nit

**Comment by erikalogie** [comment]
> 

Ok, I'll ask Sarah/Emily what they want here and get back to you.

**Comment by flagrede** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**